apps/web/src/server/resources/searchResources.ts

```diff
@@ -60,16 +60,8 @@ export const countResources = async (
                  to_tsquery('french', unaccent(${searchTerm}))
           )
         AND (
-          /* Authorization*/
           /* Resource is public  */
           resources.is_public = true
-              /* Resource is private and user is creator */
-              /* Null will never match as created_by_id is not nullable */
-              OR resources.created_by_id = ${userId}::uuid
-              /* User is contributor */
-              OR resource_contributors.id IS NOT NULL
-              /* User is member of base */
-              OR base_members.id IS NOT NULL
           )
         AND (
```

Comment:
Ces conditions d'autorisations me semblent ok, le scope du ticket est de masquer les ressource en brouillon donc plutôt ajouter un gros` AND  resources.published IS NOT NULL`  non ?